marketplace_builder/pages/getting-started/users/creating-sign-up-for.liquid

```diff
@@ -0,0 +1,245 @@
+
+```yml
+developer:
+  enabled:
+    property_options:
+      default: true
+      readonly: true
+  validation:
+    presence: true
+```
+
+can be also represented as an independent form, with configuration like this:
+
+{% raw %}
+
+```yml
+...
+resource: UserProfile
+configuration:
+  enabled:
+    property_options:
+      default: true
+      readonly: true
+...
+```
+
+{% endraw %}
+
+In other words, forms are nested in each other—you can access them as 'nested forms'. In this example you want to create a User and a corresponding UserProfile at the same time. This behavior is desired if in your UI you have a page with a button that says "Sign up as a Developer". User provides first name, email, password and that's it-the user registers as a developer. 
+
+Alternatively, the UI can split the process into two separate actions. It can first ask the user to register, so the button would be just 'Sign up', and only ask after registration if the user wants to become a developer or a client. In this scenario, the first form would look like this:
+
+```yml
+---
+name: generic_sign_up
+resource: User
+redirect_to: /
+configuration:
+  email:
+  password:
+  first_name:
+    validation: { presence: true }
+---
+```
+
+and the second one would be similar to:
+
+```yml
+---
+name: become_developer_form
+resource: UserProfile
+redirect_to: /
+configuration:
+  enabled:
+    validation: { presence: true }
+    property_options:
+      default: true
+      readonly: true
+---
+```
+
+The end result would be the same. The point is, that with Platform OS, you have the power of creating multiple resources within one form submission-which might be more complex in terms of configuration, but it can greatly simplify UX.
+
+#### Rendering inputs
+
+The second part of the form is the actual html that will be presented to the user. There are few things that you will want to include in every form.
+
+* **form tag**: {% raw %}{% form %}{% endraw %} form tag renders the `<form>` html tag with `action` set based on the `resource` provided in configuration. It also adds a bunch of hidden inputs. The most important are CSRF token (called `authenticity-token`) and `_method` - depending whether you want to create or update the resource, it will be either `post` or `patch`. In case you want to allow users to destroy their resources, you would need to manually pass `method` as argument to tag with value `delete`. Another set of hidden inputs are `resource_id` - the one that you passed in `render_form` if any, and `slug`, `slug2`, `slug3` - in case of validation errors, you most likely will need those to be available. Please note: If you want to invoke our write API manually (for example via AJAX or even from third-party API) we recommend to take a look at [full API Endpoints documentation](/reference/api-endpoints).
```

Comment:
@diana-lakatos can you turn on word-wrap at something around 100 chars? Its hard to scroll horizontally ;-)